lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
not sure if we even should be using it, in its current limited usage this could be added expilcitly with search_params, its just the name search_params is a bit unlucky here